src/models/github-server-app.ts

```diff
@@ -174,3 +186,22 @@ GitHubServerApp.init({
 		allowNull: false
 	}
 }, { sequelize });
+
+const encryptSecretFields = async (app: GitHubServerApp, fields?: string[]) => {
```

Comment:
Per @mboudreau 's comments, I extract this. But I am having trouble moving this up into the base class.

Reply:
yeah, this is a bit asymmetric with this encryption logic in `GitHubServerApp`, but the rest in `EncryptionModel`.

How about moving this function to `EncryptionModel` and replacing the `app` parameter with a parameter of type `EncryptionModel`. We'd have to add an abstract function to `EncryptionModel` then that returns all the secret fields.